test/app-tests/loading.js

```diff
@@ -343,6 +346,40 @@ describe('loading:', function () {
             }).done(done, done);
         });
 
+        it("registers correctly with a Riot Team Server", function() {
+            sdk.setFetch(httpBackend.fetchFn); // XXX: ought to restore this!
```

Comment:
well, I'm not, but it's only used in RtsClient and autocomplete/DuckDuckGoProvider.js for now. The thinking is that we can add more things to sdk.setFetch if and when it is useful.